model/src/test/kotlin/utils/FindingCurationMatcherTest.kt

```diff
@@ -261,5 +261,43 @@ class FindingCurationMatcherTest : WordSpec() {
                 )
             }
         }
+
+        "Given two curations matching a single finding, applyAll" should {
+            "return two findings with the respective curation applied" {
+                val findings = listOf(
+                    LicenseFinding(
+                        license = "MIT",
+                        location = TextLocation(path = "some/path", startLine = 1, endLine = 1)
+                    )
+                )
+                val curations = listOf(
+                    LicenseFindingCuration(
+                        path = "some/path",
+                        detectedLicense = "MIT",
+                        reason = INCORRECT,
+                        concludedLicense = "MIT-old-style"
+                    ),
+                    LicenseFindingCuration(
+                        path = "some/path",
+                        detectedLicense = "MIT",
+                        reason = INCORRECT,
+                        concludedLicense = "LicenseRef-scancode-waranty-disclaimer"
```

Comment:
Should be "warranty". Also, it seems ScanCode does not define this LicenseRef, can you instead use one that exists?

Reply:
Without that type the license ref indeed exists in ScanCode. Anyhow I just used a different one which obviously exists.